target/linux/apm821xx/dts/meraki-mx60.dts

```diff
@@ -92,23 +92,95 @@
 
 &EMAC0 {
 	status = "okay";
-	phy-handle = <&phy0>;
 
 	mdio {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		phy0: ethernet-phy@0 {
-			compatible = "ethernet-phy-id004d.d034";
+
+		switch0@10 {
+			compatible = "qca,qca8327";
+			#address-cells = <1>;
+			#size-cells = <0>;
+
 			reg = <0>;
```

Comment:
caught this during testing. 

reg should match the unit-address. Please set reg to 0x10.

Reply:
according to @Ansuel , it's ignored by the driver.

Reply:
I fixed this anyway.